nodejs/week3/src/backend/api/reservations.js

```diff
@@ -0,0 +1,57 @@
+const express = require("express");
+const app = express();
+const router = express.Router();
+const knex = require("../database");
+const { returning } = require("../database");
+
+router.get("/", async (request, response) => {
```

Comment:
Great job! ^^

Like in the above file, you only need to add some parameter validation and proper error handling to communicate different error states to the client.